Product/Production/Gateway/AdminGUIWebservices/src/main/java/gov/hhs/fha/nhinc/admingui/dashboard/DashboardStatusWebservice.java

```diff
@@ -96,13 +99,19 @@ public LogEventSimpleResponseMessageType listErrorLog(ListErrorLogRequestMessage
     @Override
     public LogEventSimpleResponseMessageType viewErrorLog(ViewErrorLogRequestMessageType arg0) {
         LogEventSimpleResponseMessageType response = new LogEventSimpleResponseMessageType();
-        response.getEventLogList().add(buildLogEventType(4, "{'Json-detail':'full-view'}", "CustomException"));
+        response.getEventLogList()
+        .add(buildLogEventType(4,
```

Comment:
Are we hardcode response in here?

Reply:
These were stubbed so @vdmehta06 and @trantang  could work independently from one another. CONN-2281 will fix this, and Tran is working on that now.